target/linux/ramips/rt305x/base-files/etc/board.d/01_leds

```diff
@@ -66,6 +66,12 @@ asiarf,awapn2403)
 dlink,dcs-930l-b1)
 	ucidef_set_led_netdev "wifi" "WiFi" "$boardname:blue:wps"
 	;;
+dlink,dir-506l)
+	ucidef_set_led_gpio "lo_bat" "Low battery" "${boardname}:red:status" "10" "0"
```

Comment:
What's that supposed to do?

Reply:
Device have battery, this rule connect internal gpio "button" (event?) to red status led to show that battery have low power.
Also ```charging``` and ```on_battery``` available, but I do not know how to make green led blink on charging.

Reply:
boardname can be dropped here (see comment for LEDs in DTS).